test/models/journeyDomain/DocumentDomainSpec.scala

```diff
@@ -233,9 +293,9 @@ class DocumentDomainSpec extends SpecBase with ScalaCheckPropertyChecks with Gen
 
     "can be read from user answers" in {
       forAll(documentGen, nonEmptyString, arbitrary[Int], arbitrary[PackageType], arbitrary[Metric], arbitrary[BigDecimal]) {
-        (document, referenceNumber, goodsItemNumber, packageType, metric, quantity) =>
+        (document, referenceInformation, goodsItemNumber, packageType, metric, quantity) =>
```

Comment:
Yes, I had to opt for this approach because the forAll method only accepts 6 parameters at most, the other approach will be to declare a separate val for the additional information outside the forAll method. What do you think?